src/games/strategy/triplea/attachments/TriggerAttachment.java

```diff
@@ -1871,9 +1888,18 @@ public static void triggerTerritoryPropertyChange(final Set<TriggerAttachment> s
             final TerritoryAttachment attachment =
                 TerritoryAttachment.get(aTerritory, t.getTerritoryAttachmentName().getSecond());
             if (attachment == null) {
-              throw new IllegalStateException("Triggers: No territory attachment for:" + aTerritory.getName()); // water territories may not
-                                                                                                                // have an attachment, so
-                                                                                                                // this could be null
+              throw new IllegalStateException("Triggers: No territory attachment for:" + aTerritory.getName()); // water
```

Comment:
The one word per line end of line comment is a bit gnarly, it is hard to read and does not help so much. Kindly just move it to the preceding line so it can be read as a single sentence.